docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,102 @@
+# VisualDL Web app Develop
+
+VisualDL has two three components.
+1. The Python/C++ SDK that logs the data during training.
+1. The single page client side app that visualized training data.
+1. The server(powered on Flask) that parses the saved data and delivers to the client side app for display.
+
+The server provided the needed data and pass to the client side to draw scalars or histograms. The users can visualize the training progress and make a better network architecture evaluation.
```

Comment:
This point "The server provided the needed data and pass to the client side to draw scalars or histograms." is kind of repeat of the previous statement.  Can probably remove.